packages/iov-bns-governance/src/index.ts

```diff
@@ -0,0 +1,137 @@
+import { As } from "type-tagger";
+
+import { Address, Amount, ChainId, PostTxResponse } from "@iov/bcp";
+import { BnsConnection, Electorate, Fraction, Proposal, VoteOption } from "@iov/bns";
+import { UserProfile } from "@iov/keycontrol";
+
+type CommitteeId = number & As<"committee">;
+
+/* Constants to be finalised once the committees are actually on-chain */
+export const COMMITTEE_IDS = {
+  ["testnet" as ChainId]: {
+    GOVERNING_BOARD: 1 as CommitteeId,
+    ECONOMIC_COMMITTEE: 2 as CommitteeId,
+    VALIDATOR_COMMITTEE: 3 as CommitteeId,
+    TECH_COMMITTEE: 4 as CommitteeId,
+  },
+};
+
+export enum ProposalType {
+  ADD_COMMITTEE_MEMBER,
```

Comment:
can we use PascalCase (as recommended by the TypeScript authors) for enum values?